addon/components/bourbon-search-select-field.js

```diff
@@ -21,7 +21,7 @@ export default Component.extend(SelectMixin, {
     if (this.get("label") !== null && this.get("value") !== null) {
       this.inputValueObserver();
     } else if (this.get("value")) {
-      this.setLabel(this.get('value'));
+      this.setLabel(this.get("value"));
```

Comment:
Why the change? It shouldn't matter in this context.

Reply:
@mshappe are you talking about the single quote to double quote?  My editor usually fixes this when that line is touched.

Reply:
Probably just for consistency, though I do still see some single quotes being used. Might want to establish a code style guide, even if just a de facto one.

Reply:
I like the idea of following the Airbnb style guide which suggests single quotes.  Will update the repo to use single quotes and update the PR accordingly.  

https://github.com/airbnb/javascript#strings--quotes